server/main/static/main/common.css

```diff
@@ -1456,3 +1456,16 @@ summary.pageDivider {
 h1.edd-nav-title {
     padding: 0.2em;
 }
+
+.centered-form {
```

Comment:
These rules define the behavior for the vertically and horizontally centered forms: sign up, login, etc. It avoids pulling in the similar existing rules in `login.css` which has a lot of other stuff that we don't need. Over time the login form can be migrated to use this class instead, leaving `login.css` to just contain the login-specific bits.